_includes/banner.html

```diff
@@ -3,56 +3,38 @@
     <div class="usa-banner__header">
```

Comment:
This should be (and always should have been, AFAIK) a `<header>` element:

```suggestion
    <header class="usa-banner__header">
```

Reference: https://designsystem.digital.gov/components/banner/

Needs corresponding changes below for the closing tag.